.github/composite-actions/compile-antlr/action.yml

```diff
@@ -0,0 +1,22 @@
+name: 'Compile ANTLR'
+
+runs:
+  using: "composite"
+  steps:  
+
+    - name: Compile ANTLR
+      run: |
+        ANTLR_VERSION=4.9.3
+        
+        cd contrib/babelfishpg_tsql/antlr/thirdparty/antlr/
+        sudo cp "antlr-$ANTLR_VERSION-complete.jar" /usr/local/lib
+        cd ..
```

Comment:
originally we were downloading the zip to  ~/work/babelfish_extensions since this cd command used to be in a new step, thirdparty is maybe a weird place to download it to, but if the tests are running still anyways this probably isn't a huge deal.